packages/cli/src/commands/apps/stacks/index.ts

```diff
@@ -34,18 +32,17 @@ export default class StacksIndex extends Command {
 
     const app = appResponse.body
     const stacks = stackResponse.body
-    const sortedStacks = _.sortBy(stacks, 'name')
+    const sortedStacks = stacks.sort((a, b) => (a.name ?? '').localeCompare(b.name ?? ''))
```

Comment:
Thank you for replacing this. We have so many uses of lodash's sortBy, I'd love to replace them all if we could. Someday.